client/ayon_core/hosts/max/api/lib.py

```diff
@@ -281,9 +282,9 @@ def reset_frame_range(fps: bool = True):
             scene frame rate in frames-per-second.
     """
     if fps:
-        project_name = get_current_project_name()
-        project_entity = ayon_api.get_project(project_name)
-        fps_number = float(project_entity["attrib"].get("fps"))
+        task_entity = get_current_task_entity()
+        task_attributes = task_entity["attrib"]
+        fps_number = float(task_attributes.get("fps"))
```

Comment:
The `.get` wouldn't help much since `float(None)` would fail anyway:

```
float(None)
# Error: float() argument must be a string or a real number, not 'NoneType'
# # Traceback (most recent call last):
# #   File "<maya console>", line 1, in <module>
# # TypeError: float() argument must be a string or a real number, not 'NoneType'
```
Also, these attributes should really exist in AYON entities and as far as I know are not optional:
```suggestion
        fps_number = float(task_attributes["fps"])
```